javaslang-test/src/main/java/javaslang/test/Gen.java

```diff
@@ -424,3 +413,27 @@
         return f.apply(this);
     }
 }
+
+final class GenModule {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.